content/eu/main.md

```diff
@@ -37,7 +37,7 @@ subtitle: The homepage of the European Galaxy community
       <div class="text-center">
         <img src="/images/undraw-illustrations/communities.svg" alt="communities" height="100">
         <br><br><br>
-        <a href="./subdomains.md" class="btn btn-primary">Check all the subdomains</a>
+        <a href="/subdomains/index.md" class="btn btn-primary">Check all the subdomains</a>        
```

Comment:
This needs to be just `/subdomains/`. Markdown filenames never appear in urls.

Reply:
hmm that doesn't find it either, I don't understand why

Reply:
oh found it, it should be `/eu/subdomains/`